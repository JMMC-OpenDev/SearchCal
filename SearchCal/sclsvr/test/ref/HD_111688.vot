version https://git-lfs.github.com/spec/v1
oid sha256:ddfdedd4abe0679de09d99d46a953d02c1db8463384e9841c86929367a83d24a
size 278359
