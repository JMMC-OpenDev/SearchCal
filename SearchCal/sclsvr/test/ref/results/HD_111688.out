version https://git-lfs.github.com/spec/v1
oid sha256:59c9458a6b9b7a8c9db01ee9ea1618a68e770b61a45c83792e42a703ca3d3994
size 127442
